handbook/engineering/web/frontend-platform/index.md

```diff
@@ -84,6 +84,15 @@ We plan and track our day-to-day work on our [Kanban board](https://github.com/o
 
 The team makes use of [tracking issues](../../tracking_issues.md) for tracking progress on the implementation of new features. The teammates should ensure that a tracking issue is created when starting work on features that are expected to take longer than a few days (or require multiple PRs) to deliver.
 
+#### Getting cross-team feedback on RFC
+
+- Create an issue for the RFC tracking on our [Kanban board](https://github.com/orgs/sourcegraph/projects/144).
+- [Lock conversation](https://docs.github.com/en/github/building-a-strong-community/locking-conversations): RFC should be discussed in the Google doc.
+- Add _rfc/tracking_ and _team/frontend-platform_ labels to the RFC issue.
+- Once RFC is ready for review, move it to the _In review_ column and assign _sourcegraph/frontend-devs_ or individual developers to the issue.
+- Github integration will notify _@web_ in #frontend-platform-rfc-feed that the RFC is ready for review.
```

Comment:
Why the whole web org specifically? Shouldn't it either be all frontend devs, or only frontend platform?